.travis.yml

```diff
@@ -2,6 +2,9 @@ services:
   - postgresql
 before_script:
   - psql -c 'CREATE DATABASE test;' -U postgres
+  - psql -c 'CREATE DATABASE users;' -U postgres
+  - psql -c 'CREATE DATABASE phantom1;' -U postgres
```

Comment:
make sure that only the testing database is created during testing
so you can remove these